.ci/hardware.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+set -e -v
+
+NAME=$0
+COMMAND=$1
+
+if [[ "$COMMAND" == "install" ]]; then
+    pip install coverage  # needed for the uploading in after_success
+    openssl aes-256-cbc -K $encrypted_e0365add3e93_key -iv $encrypted_e0365add3e93_iv -in .ci/travis_rsa.enc -out ~/.ssh/id_rsa -d
+    chmod 600 ~/.ssh/id_rsa
+    echo -e "$INTELHOST_INFO" >> ~/.ssh/config
+    ssh -o StrictHostKeyChecking=no loihihost "echo 'Connected to loihihost'"
+    scp -r . loihihost:/tmp/nengo-loihi-$TRAVIS_JOB_NUMBER
+elif [[ "$COMMAND" == "script" ]]; then
+    ssh loihihost << EOF
+        sh /etc/profile
+        sh ~/.bashrc
+        cd /tmp/nengo-loihi-$TRAVIS_JOB_NUMBER
+        conda create -y -n travis-ci-$TRAVIS_JOB_NUMBER python=3.5.2 scipy
+        source activate travis-ci-$TRAVIS_JOB_NUMBER
+        pip install -e .[tests]
+        pip install ~/travis-ci/nxsdk-0.7.5.tar.gz
+        SLURM=1 coverage run -m pytest --target loihi --no-hang -v --duration 50 --color=yes && \
```

Comment:
Is `duration` the same as `durations`? It's listed as the latter in `pytest --help`.

Reply:
Yeah, apparently either one works (I didn't even notice).  I'll change it to durations just for consistency though.

Reply:
`pytest` is like that. I always use `pytest --plot` to make plots, but the name we actually gave the parameter is `--plots`.